Source/src/modules/ModuleResources.h

```diff
@@ -32,7 +32,7 @@ namespace Hachiko
         std::vector<UID> ImportAssetFromAnyPath(const std::filesystem::path& path, bool force = false);
         std::vector<UID> CreateAsset(Resource::Type type, const std::string& name) const;
         void LoadAsset(const std::string& path);
-        GameObject* InstantiatePrefab(UID prefab_uid, GameObject* parent);
+        std::vector<GameObject*> InstantiatePrefab(UID prefab_uid, GameObject* parent, unsigned n_instances);
```

Comment:
Also maybe instead of making this return `std::vector` it can return `GameObject*` that is allocated like `new GameObject[n_instances];` so that the API is same and cleaner for when `n_instances` is 1.

Reply:
I like the dumb dumb simplicity of having everything managed by a vector, maybe we could have a single instance alternative that returns a single gameobject*